File: python_work/220707/ex02.py
'''
    문제1 0부터 시작해서 값을 하나씩 증가시키며 9까지 출력을 보이는 코드를 While 루프를 기반으로 작성해보자

    문제 2
    9에서부터 값을 하나씩 감소시키며 0까지 출력을 보이는 코드를 while 루프를 기반으로 작성해보자
'''


def doA():
    num = 0
    while num < 10:
        print("num = ", num)
        num += 1


def doB():
    num = 9
    while num > -1:
        print("num = ", num)
        num -= 1
doA()
doB()

File: java_work/ex0715/src/ex0715/Ex02.java
package ex0715;


interface A{
	public void doA();	// 재정의 해서 사용해야 된다. 오버라이드 해서 사용해야한다.
}

interface B{
	public void doB();
}

public class Ex02 implements A,B{
	public void doA() {
		
	}
	public void doB() {
		
	}
}
